### PR TITLE
AmazonPay setup guide changes to support new Integration Central pages

### PR DESCRIPTION
**Pull request by RayAnaya**
## Purpose of this pull request

This pull request (PR) addresses changes in Amazon Pay's "Seller Central" web site, which affect how merchants obtain and copy the necessary credentials for their Magento and Amazon Pay integration (surfaced through our CBE).  It contains updated screen-shots and directions, but does not ADD or REMOVE any functionality and is not related to any specific Magento version, or Amazon Pay plugin version.

If Magento Commerce, please specify:

- [X] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/user-guide/payment/amazon-pay-seller-central.html

## Additional information

As per our most recent product updates, Amazon Pay has moved **where** a merchant may find their access and secret keys.  We are proposing the attached changes in order to keep documentation current.  This is independent of our core-bundled-extension and our integration with Magento.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by RayAnaya** [comment]
> @RayAnaya Thank you for this update! Please sign the Adobe CLA so that we can process this PR: http://opensource.adobe.com/cla.html

Done and done.  I'm having my team do one final review of the changes before I will mark the PR as ready for review.

**Comment by leslietilling** [comment]
@RayAnaya  awesome!  We will process as soon as you are ready.

**Comment by RayAnaya** [comment]
@leslietilling All good to go!  Apologies for the additional commit change, the team and I realized we hadn't redacted some information on one of the screenshots :)

File: src/payment/amazon-pay-seller-central.md [hunk 1]
```diff
@@ -26,26 +25,26 @@ To run test transactions from the Amazon Pay Sandbox, you must copy the keys fro
...
    You will need them to complete the Amazon Pay configuration for Sandbox Mode in the Magento Admin.
 
    {:.bs-callout-info}
    Your keys are confidential and should not be shared.
 
-   ![Your Credentials]({% link images/images/payments-amazon-pay-your-keys.png %}){: .zoom}
-   _Your Credentials_
-
-1. Close the box.
+1. Use these keys to complete your setup inside of the Magento Admin Portal.
```
Comment (leslietilling) [inline src/payment/amazon-pay-seller-central.md:50]:
```suggestion
1. Use these keys to complete your setup inside of the Magento Admin.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**